Article-Front-End-Interview-Questions.md

```diff
@@ -0,0 +1,78 @@
+# Interview questions for Junior Front-End Web Developers
+
+The purpose of this article is to familiarize fresh Front-End Web Developers with text book questions. If you are just getting into the developer world there is a good chance you will feel nervous when interviewing for a developer's positon. Because of this, some relatively basic questions could feel more difficult than how they actually are. A good counter for this scenario is to be properly prepared. Without further ado, here are a few questions that you could encounter:
+
+1.	In the context of CSS styling can you explain what does display-style: inline-block stand for? 
+
+
+
+2.	If you have an `<ul>` element with several `<li>` elements, how would you go about setting up these elements in a row by using HTML and CSS? 
+
+
+3.	In CSS styling when would you use `float`s? How do they work? 
+
+
+4.	When would you use: `clear: both;`?
+
+
+5.	What is Bootstrap? When would you use it? How does its grid system work? 
+
+
+6.	Explain the notion of responsive design and how it works? Use an example if you like.
+
+
+7.	What is a DOM? What do you know about it? 
+
+
+8.	What is ECMAScript?
+
+
+9.	What is jQuery and what is it commonly used for? 
+
+
+10.	What is the difference between `==` and `===` in JS? 
+
+
+11.	Can you enumerate all JavaScript data types?
+
+
+12.	What is a callback function in JS and when would you use one? 
+
+
+13.	What is the difference between `undefined` and `null`? 
+
+
+14.	What is an async response? 
+
+
+15.	What does JSON stand for? What is it used for?
+
+
+16.	How would you explain an API request?
+
+
+17.	What APIs have you worked with?
+
+
+18.	How would you send out/convert to JSON format?
+
+
+19.	What is the difference between `GET` and `POST` when specifying an AJAX request?
+
+
+20.	What is the meaning of PSD? 
```

Comment:
As I said in earlier diff - this is out of the blue. A lot of Front-End Developers don't work with PSD file formats. Either remove the question, or phrase it in a way that helps.